lib/cipherstash/ordered_string_test_generator.rb

```diff
@@ -0,0 +1,98 @@
+require "securerandom"
+require "json"
+
+require_relative "./index"
+
+module CipherStash
+  # Generates test cases for orderable strings that can be used in automated tests in other
+  # clients (e.g. StashRS). Test cases are written to JSON files.
+  #
+  # Example usage in StashRS:
+  # https://github.com/cipherstash/cipherstash-rs/blob/00ff66d712e6f36a89acfa731b680a69789647cf/packages/cipherstash-client/src/indexer/mapping_indexer.rs#L537-L570
+  #
+  # @private
+  class OrderedStringTestGenerator
```

Comment:
I wasn't too sure where this class/file should live. I'm definitely open to suggestions if this seems out of place.

Reply:
It's more of a client-specific thing, so perhaps `CipherStash::Client::OrderedStringTestGenerator`.  I agree with you that it does feel a tiny bit "weird" no matter where I can think of it to put it.  Almost like it shouldn't be in this repo at all -- maybe a "helper" script of some sort in StashRS?  It's not something we'll need to run often, presumably.

That being said, I'm not strongly opposed to it living in here in some out-of-the-way corner.  It has to go *somewhere*, after all; it's too handy not to have.